core-bundle/src/Routing/Matcher/LanguageFilter.php

```diff
@@ -22,34 +22,31 @@
  */
 class LanguageFilter implements RouteFilterInterface
 {
-    /**
-     * @var bool
-     */
-    private $prependLocale;
-
     /**
      * @internal Do not inherit from this class; decorate the "contao.routing.language_filter" service instead
      */
-    public function __construct(bool $prependLocale)
+    public function __construct()
```

Comment:
I think you can remove the constructor completely and move the `@internal` tag to the class level phpDoc comment.